src/Pester.ps1

```diff
@@ -255,17 +255,26 @@ function Has-Flag {
 function Invoke-Pester {
     [CmdletBinding(DefaultParameterSetName = 'Simple')]
     param(
-        [Parameter(Position = 0, Mandatory = 0,  ParameterSetName = "Simple")]
+        [Parameter(Position = 0, Mandatory = 0, ParameterSetName = "Simple")]
+        [Parameter(Position = 0, Mandatory = 0, ParameterSetName = "Legacy")]  # deprecated
+        [Alias("Script")] # deprecated
```

Comment:
If the comments said `Legacy` or `Legacy ParameterSet` instead it would be easier to infer that it refers to the `Legacy` parameter set

Reply:
done